<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdventureMenu</class>
 <widget class="QWidget" name="AdventureMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{border-image: url(:/processed/mmAdventureMenu_Fixed.png);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="advPlayButton">
    <property name="geometry">
     <rect>
      <x>544</x>
      <y>442</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="advMainMenuButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>436</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnA1">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>300</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
